x-pack/solutions/security/packages/kbn-scout-security/index.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export { test, spaceTest } from './src/playwright';
+
+// re-export from @kbn/scout
+export { expect, tags, createPlaywrightConfig, createLazyPageObject } from '@kbn/scout';
```

Comment:
FYI - I'm seeing this ESLint rule error - are you seeing it too? It looks like something that the Operations team set up, though I'm not sure how to fix this.

![image](https://github.com/user-attachments/assets/f1c7ea06-5b59-4391-9a7f-63ea72d1f08a)

Reply:
I don't see those errors and no lint error is thrown on the CI

Reply:
Thanks for letting me know Gloria. Good to know, must be something on my side then. Cheers!